jest.config.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/en/configuration.html
+ */
+
+module.exports = {
```

Comment:
Could we remove all the comments within the `module.exports` ? I like having the docs link above, but the comments within make it a bit hard to read IMO